sunbird-cassandra-migration/cassandra-migration/src/main/resources/db/migration/cassandra/V1.98_cassandra.cql

```diff
@@ -1,3 +1,8 @@
 INSERT INTO sunbird.role(id, name, rolegroupid, status) VALUES('COURSE_CREATOR', 'Course Creator', ['COURSE_CREATOR'], 1);
 INSERT INTO sunbird.role_group(id, name, url_action_ids) VALUES('COURSE_CREATOR', 'Course Creator', ['courseCreator']);
-INSERT INTO sunbird.email_template(name,template) VALUES('verifyPhoneOtpTemplateWard', 'Your ward has requested for registration on $installationName using this email ID. Use OTP $otp to agree and create the account. This is valid for $otpExpiryInMinutes minutes only.');
+
```

Comment:
If once any cassandra script file is executed in any environment , on updating any scripts in this file won't be executed again. Code looks for another +1 version. Please move these scripts to new V1.99_cassandra.cql file.

Reply:
Resolved as per the comments